index.html

```diff
@@ -223,6 +223,52 @@ <h3>
           </ul>
         </div>
       </section>
+      <section>
+        <p class="contact-paragraph">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          id="contact-form"
+          action="https://formspree.io/f/xknlnqre"
+          method="post"
+        >
```

Comment:
> A user cannot submit the form if one or more of the fields is empty.
- - You have done great so far. :+1: :+1: But there are some requirements you still need to implement before moving to the next milestone. :+1: 
- [ ] As stated in the project requirement, all input fields needs to be filled before a user can submit a form. Currently that's not the case in your project. I'll suggest you add the `required` attribute to all the inputs in order to meet this requirement. :+1: :wave: